src/mjolnir/valhalla_add_landmarks.cc

```diff
@@ -40,18 +41,7 @@ int main(int argc, char** argv) {
     return EXIT_FAILURE;
   }
 
-  auto pt = valhalla::config();
-
-  // configure logging
-  auto logging_subtree = pt.get_child_optional("mjolnir.logging");
-  if (logging_subtree) {
-    auto logging_config =
-        valhalla::midgard::ToMap<const boost::property_tree::ptree&,
-                                 std::unordered_map<std::string, std::string>>(logging_subtree.get());
-    valhalla::midgard::logging::Configure(logging_config);
```

Comment:
is this inside valhalla::config() now i expected it was in the parsing args place

Reply:
ah i see its already in the parsing args spot just was redundant!